x/mongo/driver/operation.go

```diff
@@ -446,7 +449,20 @@ func (op Operation) Execute(ctx context.Context, scratch []byte) error {
 			if e := err.(Error); retryable && op.Type == Write && e.UnsupportedStorageEngine() {
 				return ErrUnsupportedStorageEngine
 			}
-			if retryable && tt.Retryable() && retries != 0 {
+
+			connDesc := conn.Description()
+			var retryableErr bool
+			if op.Type == Write {
+				retryableErr = tt.RetryableWrite(connDesc.WireVersion)
+				// Add a RetryableWriteError label for retryable errors from pre-4.4 servers
+				if tt.HasErrorLabel(NetworkError) || (retryableErr && connDesc.WireVersion != nil && connDesc.WireVersion.Max < 9) {
```

Comment:
Ah this makes sense now... for >=4.4, the network errors won't be labeled correctly. Can you update the comment to mention both cases? My confusion yesterday was because the comment only talks about the second case.

Reply:
done